Makefile.am

```diff
@@ -35,3 +35,25 @@ MAINTAINERCLEANFILES = \
 	aclocal.m4 compile config.guess config.sub \
 	configure depcomp install-sh ltmain.sh     \
 	Makefile.in missing
+
+if HAVE_RPMBUILD
+rpm: build/$(PACKAGE).spec
+	rm -rf ./va/.rpmbuild
+	mkdir -p ./va/.rpmbuild/BUILD \
+	./va/.rpmbuild/SPECS \
+	./va/.rpmbuild/SOURCES \
+	./va/.rpmbuild/SRPMS \
+	./va/.rpmbuild/RPMS/noarch \
+	./va/.rpmbuild/tmp
+	git archive --format=tar.gz --prefix=$(PACKAGE)-$(VERSION)/ -o ./$(PACKAGE)-$(VERSION).tar.gz HEAD
```

Comment:
if git is not installed, Does it still work?

Reply:
Ok, changing to use "make dist-gzip" instead of git

Reply:
"make dist-gzip" looks like it fails twice, then succeeds. Seems to be generating files related to wayland. Looking into how to make it work on the first try

Reply:
Ok, I switched to using tar. Do you think it is safe to assume that tar is available?